### PR TITLE
Add node-code-formatter

### PR DESCRIPTION
**Pull request by MarvinJWendt**
Add [node-code-formatter](https://github.com/marketplace/actions/node-code-formatter).

## Readme:

# Node Code Formatter

[GitHub Action](https://github.com/marketplace/actions/node-code-formatter)
[GitHub license](https://github.com/MarvinJWendt/run-node-formatter/blob/master/LICENSE)
[GitHub issues](https://github.com/MarvinJWendt/run-node-formatter/issues)
[GitHub stars](https://github.com/MarvinJWendt/run-node-formatter/stargazers)

> Automatically formats your code!

## Automatically format pull requests

_Never tell your users to format their code, as we do it on the fly!_

image

## Usage :pencil2:

1. Create a `formatter.yml` file in `.github/workflows/`
2. Paste this code into the file:

```yml
on: push
name: Node Code Formatter
jobs:
  lint:
    name: Node Code Formatter
    runs-on: ubuntu-latest
    steps:
    - name: Node Code Formatter
      uses: MarvinJWendt/run-node-formatter@1.2.1
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

3. Commit the file :twisted_rightwards_arrows:

## Setup formatter scripts :clipboard:

Simply put your code formatter into a script named `format` or `lint` in your `package.json` (Yarn only supports a `lint` script at the moment).

**Make sure that your code formatter is a dependency of your module!**

### StandardJS

```json
...
"scripts": {
    "format": "standard --fix"
  }
```

### Prettier

```json
...
"scripts": {
    "format": "prettier"
  }
```

### CODE REVIEW

**Comment by boyney123** [comment]
Thanks for the contribution @MarvinJWendt 🙏 👍 

Merged and also added you as a contributor on the main README.

Thanks
